### PR TITLE
Allow support for alternate AWS profiles.

### PR DESCRIPTION
**Pull request by seanstar12**
## What Changed & Why

Added variable and condition check to allow the swapping of AWS profiles.
## PR Checklist
- [ ] Add tests -- Config is already tested for errors. Error(s) would occur in AWS-SDK.
- [x] Add documentation
- [x] Prefix documentation-only commits with [DOC]

### CODE REVIEW

**Review by achambers — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -83,6 +83,13 @@ This requires both `accessKeyId` and `secretAccessKey` to be defined.
 
 *Default:* `undefined`
 
+### profile
```
Comment (achambers) [inline README.md:86]: Could you please change the commit message that contains this documentation change to begin with `[DOC]` ?

**Review by achambers — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (seanstar12) [reply README.md:86]: Change has been made.

**Review by achambers — APPROVED** [review]

**Comment by achambers** [comment]
Thanks @seanstar12 . I'll try and get this released shortly.

**Comment by seanstar12** [comment]
@achambers Thanks! Any word on when it will be pushed to npm?

**Pull request merged**